### ISSUE TITLE
unable to build on centos 6.5

### ISSUE THREAD

**Issue by jacekjaros** [UlricE/pen#12]
my issue:
```
# ./configure --prefix=/opt/pen/
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
...
checking for main in -lssl... no
checking for EC_KEY_new_by_curve_name... no
checking for GeoIP_country_code_by_addr in -lGeoIP... no
./configure: line 6564: syntax error near unexpected token `newline'
./configure: line 6564: `  yes:no:'
```

**Comment by UlricE** [comment]
Added a pair brackets to the part of configure.ac that seemed to be causing the issue. It now builds for me on Centos 6.6.

**Comment by UlricE** [comment]
Closing this, since the above fix is in 0.30.0.
